airmencoders/Logging-iOS
# Feature/summary cards

### Discussion :speech_balloon: 
Splitting improvements to follow-on Issue

### What this PR does :thinking: 
Rearranges summary information for sub-sortie pages

### Resolves :white_check_mark: 
[PBS-2407](https://jira.il2.dso.mil/browse/PBS-2407)

### Testing Items :microscope: 
- Navigate to sortie
- Verify previous functionality via new summary cards

### Notes :writing_hand: 

### UI Changes :eye: 

### Resources :books:
